resources/database.rb

```diff
@@ -63,15 +63,16 @@
   include PostgresqlCookbook::Helpers
 
   def database_exists?(new_resource)
-    sql = %(SELECT datname from pg_database WHERE datname='#{new_resource.name}')
+    sql = %(SELECT datname from pg_database WHERE datname='#{new_resource.database}')
 
-    exists = %(psql -c "#{sql}" postgres)
+    exists = %(psql -c "#{sql}")
+    exists << " -U #{new_resource.user}" if new_resource.user
     exists << " --host #{new_resource.host}" if new_resource.host
     exists << " --port #{new_resource.port}" if new_resource.port
-    exists << " | grep #{new_resource.name}"
+    exists << " | grep #{new_resource.database}"
 
     cmd = Mixlib::ShellOut.new(exists, user: 'postgres')
     cmd.run_command
-    cmd.exitstatus == 0
+    cmd.exitstatus.zero?
```

Comment:
Cookstyle doesn't like this, even though I prefer the style (and use this in other Ruby projects!)